opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -178,11 +180,13 @@ def create(
         resource = _DEFAULT_RESOURCE
 
         otel_experimental_resource_detectors = environ.get(
-            OTEL_EXPERIMENTAL_RESOURCE_DETECTORS, "otel"
+            OTEL_EXPERIMENTAL_RESOURCE_DETECTORS, "otel,os"
```

Comment:
```suggestion
            OTEL_EXPERIMENTAL_RESOURCE_DETECTORS, ""
```

While we are at it we can make it an empty string and rely on the following lines to add the defaults so next time we need to remember to update them in just one place

Reply:
Watch out:

```
In [1]: "".split(",")
Out[1]: ['']
```